translator.py

```diff
@@ -0,0 +1,129 @@
+
+            # This "if" checks if a 2:1 type of trade succeeded
+            target_beloww = "%.2f" %((Trigger - 2 * (Stop - Trigger)))
+            two_one_partial = input("Did you reach your 2:1 target of " + str(target_beloww) + str("?"))
+            if two_one_partial == "no":
+                Executed_stop = float(input("When did Stop execute?"))
+                loss = "%.2f" % (abs(Executed_stop - Executed_trigger) * Shares)
+                print("Your 2:1 trades have lost: " + str(loss) + "$")
+                print("If you traded 1:1 100% you earned " + str(one_one_full_profit) + "$")
+                print("If you traded 1:1 75% 25% you earned " + str(one_one_partial_profit) + "$ + " + str(remainder_one_one_profit) + "$")
+            elif two_one_partial == "yes":
+                two_one_partial_profit = "%.2f" % ((Executed_trigger - (Trigger - (Stop - Trigger) * 2)) * 0.5 * Shares)
+                target_belowww = "%.2f" % (Trigger - 4 * (Stop - Trigger))
+                four_one_partial = input("Did you reach your 4:1 target of " + str(target_belowww) + str("?"))
+                if four_one_partial == "no":
+
+                    # if trade didn't make it to 1:4 it means all trades have been covered at this stage. I will print results.
+
+                    print("All trades concluded:")
+                    print("If you traded 1:1 100% you made " + str(one_one_full_profit) + "$")
+                    print("If you traded 1:1 75% 25% you made " + str(one_one_partial_profit) + "$ + " + str(remainder_one_one_profit) + "$")
+                    print("If you traded 2:1 variation you made " + str(two_one_partial_profit) + "$")
+
+                elif four_one_partial == "yes":
+
+                    # This "if" is for fully successfull 2:1 trades. I will calculate and print results.
+
+                    four_one_partial_profit = "%.2f" % ((Executed_trigger - (Trigger - (Stop - Trigger) * 4)) * 0.25 * Shares)
+                    four_one_full_profit = "%.2f" % ((Executed_trigger - (Trigger - (Stop - Trigger) * 4)) * 0.5 * Shares)
+                    remainder_four_one = float(input("Where did you cover remainder of 1:2 trade: "))
+                    remainder_four_one_profit = "%.2f" % ((Executed_trigger - remainder_four_one ) * 0.25 * Shares)
+
+                    print("All trades concluded:")
+                    print("If you traded 1:1 100% you made " + str(one_one_full_profit) + "$")
+                    print("If you traded 1:1 75% 25% you made " + str(one_one_partial_profit) + "$ + " + str(remainder_one_one_profit) + "$")
+                    print("If you traded 1:2 50% 25% 25% you made " + str(two_one_partial_profit) + "$ + " + str(four_one_partial_profit) + "$ + " + str(remainder_four_one_profit) + "$.")
+                    print("If you traded 1:2 50% 50% you made " + str(two_one_partial_profit) + "$ + " + str(four_one_full_profit) + "$.")
+
+                    clear = lambda: os.system('cls')
```

Comment:
Are you able to explain what exactly is a lambda?

Reply:
No. I asked my friends how to clear the console after each use and this is what they gave me. It worked so I left it in.

Reply:
https://www.w3schools.com/python/python_lambda.asp